Try to resolve first comment (#5)
liuhenry4428/cs6120: content/blog/2019-09-25-RecordTypes.md

Immutable data structures are easier to reason about when developing compiler optimizations. However, forcing developers to recreate the entire record every time they need to change a value poses usability challenges. We decided on sticking to immutability in part for optimizations' sake. The ability to make guarantees that a value will not change lends itself well to constant-folding, even after a function call that takes in your record as an argument. 

### Evaluation

> The two main goals of record types was to allow Bril to logically group and use related data points and to have them be valuable as a language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 

Reviewer: ```suggestion
The two main goals of record types were to allow Bril to logically group and use related data points and to have them be valuable as a language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 
```

Reviewer: ```suggestion
The two main goals of record types are to allow Bril to logically group and use related data points and to have them be valuable as a language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 
```

> The two main goals of record types was to allow Bril to logically group and use related data points and to have them be valuable as a language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 

Reviewer: ```suggestion
The two primary goals of record types are to allow Bril to logically group and use related data points and have them be valuable as a language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 
```

> The two main goals of record types was to allow Bril to logically group and use related data points and to have them be valuable as a language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 

Reviewer 2: ```suggestion
The two main goals of record types are to allow Bril to logically group and use related data points as well as to serve as a valuable language feature. These would be useful when compiling higher-level languages that utilize a similar data structure like records in OCaml or structs in C into Bril. 
```
